assets/css/styles.css

```diff
@@ -1,7 +1,7 @@
 /* Edo Font */
 @import url("http://fonts.cdnfonts.com/css/edo");
 
-.bg-img {
+.img {
```

Comment:
Remove this there is a css property called img {} make it as .img @bhavy2202

Reply:
Should I change .img to img?

Reply:
no img to .img in the bottom not this @bhavy2202